2009-05-26  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/alpha/Makefile (CFLAGS-rtld.c): add $(PIC-ccflag).
	(CFLAGS-dl-close.c): Likewise.

--- a/ports/sysdeps/alpha/Makefile
+++ b/ports/sysdeps/alpha/Makefile
@@ -35,6 +35,9 @@
 ifeq ($(subdir),elf)
 # The ld.so startup code cannot use literals until it self-relocates.
 CFLAGS-rtld.c = -mbuild-constants
+# It uses more than 64k for the small data area.
+CFLAGS-rtld.c    += $(PIC-ccflag)
+CFLAGS-dl-close.c = $(PIC-ccflag)
 endif
 
 # Build everything with full IEEE math support, and with dynamic rounding;
